app/models/micropost.rb

```diff
@@ -0,0 +1,16 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  scope :recent_posts, ->{order created_at: :desc}
+  has_one_attached :image
```

Comment:
thứ tự scope đứng sau validates nha em
```suggestion
  belongs_to :user
  has_one_attached :image
  validates ...
  scope :recent_posts, ->{order created_at: :desc}
```